ci/test

```diff
@@ -110,12 +110,11 @@ rspec() {
 rspec_audit() {
   : DOC - Runs RSpecs for the Audit engine
   
-  # Start Conjur with the audit database
-  docker-compose up --no-deps -d audit pg
+  # Start Conjur with the postgres database
+  docker-compose up --no-deps -d pg
 
-  _wait_for_pg audit
+  _wait_for_pg pg
 
-  AUDIT_DATABASE_URL=postgres://postgres@audit/postgres \
     docker-compose run -T --rm --no-deps  -w /src/conjur-server cucumber -ec "
       pwd
       ci/rspec-audit/migratedb
```

Comment:
see comment above regarding double migrations

Reply:
AUDIT_DATABASE_URL is not longer in used.